OpenEdXMobile/res/values/styles.xml

```diff
@@ -596,4 +596,14 @@
         <item name="lineHeight">@dimen/widget_margin_x_double</item>
     </style>
 
+    <style name="edX.Widget.SecondaryBorderedButton" parent="edX.Widget.Button">
```

Comment:
Please use similar styling as `edX.Widget.PrimaryActionButton` or update that as well.